cl/scrapers/tasks.py

```diff
@@ -567,51 +273,36 @@ def process_audio_file(pk):
     """
     af = Audio.objects.get(pk=pk)
     tmp_path = os.path.join("/tmp", "audio_" + uuid.uuid4().hex + ".mp3")
-    av_path = get_audio_binary()
-    av_command = [
-        av_path,
-        "-i",
-        af.local_path_original_file.path,
-        "-ar",
-        "22050",  # sample rate (audio samples/s) of 22050Hz
-        "-ab",
-        "48k",  # constant bit rate (sample resolution) of 48kbps
-        tmp_path,
-    ]
-    try:
-        _ = subprocess.check_output(av_command, stderr=subprocess.STDOUT)
-    except subprocess.CalledProcessError as e:
-        print(
-            "%s failed command: %s\nerror code: %s\noutput: %s\n%s"
-            % (
-                av_path,
-                av_command,
-                e.returncode,
-                e.output,
-                traceback.format_exc(),
-            )
-        )
-        raise
+    response = convert_audio(af.local_path_original_file.path)
 
-    set_mp3_meta_data(af, tmp_path)
-    try:
-        with open(tmp_path, "r") as mp3:
-            cf = ContentFile(mp3.read())
-            file_name = trunc(best_case_name(af).lower(), 72) + "_cl.mp3"
-            af.file_with_date = af.docket.date_argued
-            af.local_path_mp3.save(file_name, cf, save=False)
-    except:
-        msg = (
-            "Unable to save mp3 to audio_file in scraper.tasks."
-            "process_audio_file for item: %s\n"
-            "Traceback:\n"
-            "%s" % (af.pk, traceback.format_exc())
-        )
-        print(msg)
+    err = response["err"]
+    if not err == Timeout:
+        with open(tmp_path, "w") as mp3:
+            mp3.write(response["content"])
+        set_mp3_meta_data(af, tmp_path)
```

Comment:
Alas, everything before now was style, but I think this is feature change. I think all the eyed3 junk and metadata setting/tweaking junk should go into the transformer too. Wouldn't it be nice if we set a wav file and some metadata to the transformer and got back an MP3 with album art and good metadata that we can just save and get on with it?

Or if we want to be less monolithic about it, we could instead do two calls, one for "make an MP3" and one for "set its metdata". Either way though, I think eyed3 is a great candidate for yanking out of here.

Reply:
I agree it should go in there - it was jus so intertwined with saving code that I didn't want to do it until I had this first draft reviewed

Reply:
Excellent.

Reply:
This has been moved out of CL - and allowed for some cleanup.